third_party/blink/renderer/platform/scheduler/common/throttling/budget_pool.cc

```diff
@@ -51,12 +51,6 @@ void BudgetPool::UnregisterThrottler(TaskQueueThrottler* throttler) {
 
 void BudgetPool::RemoveThrottler(base::TimeTicks now,
                                  TaskQueueThrottler* throttler) {
-  if (!recordreplay::AreEventsDisallowed()) {
```

Comment:
Note: This Assert diverges if we keep it in, but its an allowed divergence, so we need to remove it.